libkineto/include/GenericTraceActivity.h

```diff
@@ -73,7 +73,11 @@ class GenericTraceActivity : public TraceActivity {
 
   //Encode client side metadata as a key/value string.
   void addMetadata(const std::string& key, const std::string& value) {
-    metadata_.push_back(fmt::format("\"{}\": {}", key, value));
+    if (!value.empty() && (value.find_first_not_of("-0123456789") == std::string::npos
```

Comment:
Nit: I think coding standard dictates use of curly brackets.

Reply:
Done